backends/occa/ceed-occa-operator.c

```diff
@@ -116,6 +116,7 @@ static int CeedOperatorApply_Occa(CeedOperator op, CeedVector qdata,
                                   data->BEv,data->etmp); CeedChk(ierr);
   // *************************************************************************
   ierr = CeedVectorRestoreArray(etmp, &Eu); CeedChk(ierr);
+  ierr = CeedVectorRestoreArray(qdata, (CeedScalar**)&qd); CeedChk(ierr);
```

Comment:
If I remove this line, I see correct results with the example, but with it, I get 

```
Exact mesh volume    :  2.3561944901923                             
Computed mesh volume :  0                                            
Volume error         : -2.3561944901923  
```

I have not been successful at debugging the difference, but it looks like a memory corruption issue. The array variable being restored is `qd` and is not used again, but somehow the entries of a different vector are being destroyed.